gsa/src/gmp/models/scanner.js

```diff
@@ -99,10 +99,12 @@ class Scanner extends Model {
       ret.id = element.uuid;
     }
 
-    if (hasValue(element.scannerType)) {
-      ret.scannerType = scannerTypeInt(element.scannerType);
+    // scanner.type can be one of three formats:
+    // integer 2 (for scanners), '2' (audits) and 'OPENVAS_SCANNER_TYPE' (hyperion)
+    if (isString(element.type) && element.type.length > 2) {
```

Comment:
maybe we'll have more scanner types in the future but i don't anticipate more than 99...